src/cli.js

```diff
@@ -9,11 +9,23 @@ const emitSuccess = message => console.log(green(` ✔ Sucesso: ${message}`));
 const emitError = message => console.log(red(` ✗ Erro: ${message}`));
 
 function cli(args) {
-    gemidao(args)
-        .then(() => {
-            emitSuccess(args.sms ? 'sms enviado!' : 'chamada efetuada!');
-        })
-        .catch(pipe(prop('message'), emitError));
+    if (args.bidirecional) {
+        send(args);
+        const temp = args.de;
+        args.de = args.para;
```

Comment:
Resolvi desacoplando os dados da msg, isso possibilita modificar o padrão from/to e fazer outra chamada send após gerar a call invertendo os valores, o que vocês acham?